packages/design-system-web/src/global/global.scss

```diff
@@ -5,7 +5,7 @@
 @import '~@ionic/core/css/normalize.css';
 /* @import '~@ionic/core/css/structure.css'; */
 @import '~@ionic/core/css/display.css';
-@import './typography.css';
+@import './typography.scss';
```

Comment:
Good question!

Short answer: We don't need it anymore. I change it back to css.

Long answer: @perrauda suggested to use .mdx files for the docs. In order to apply our css to the html generated from .mdx files by storybook, we needed the class .sbdocs on all selectors. Instead of adding them to each selector, I wrapped them in .sbdocs {} in scss.

Now we don't need .mdx because I managed to generate the docs automatically (using docsTemplate.tsx and docs-generator/**).